tests/chunks.test.js

```diff
@@ -304,55 +326,128 @@ describe('chunks', () => {
         {
           type: 'clientFilesCourse',
         },
+        {
+          type: 'clientFilesCourseInstance',
+          courseInstanceId,
+        },
+        {
+          type: 'clientFilesAssessment',
+          courseInstanceId,
+          assessmentId,
+        },
+        {
+          type: 'question',
+          questionId,
+        },
       ];
 
       // Generate chunks for the example course
-      let jobId = await chunksLib.generateAllChunksForCourseList(course_ids, authn_user_id);
-      await helperServer.waitForJobSequenceSuccessAsync(jobId);
+      await chunksLib.updateChunksForCourse({
+        coursePath: tempTestCourseDir.path,
+        courseId,
+        courseData: await courseDB.loadFullCourse(courseDir),
+      });
```

Comment:
This is nicer than using a job, as we'll actually get a meaningful error message if something goes wrong.